css-structure/css-structure_custom-properties/css/styles.css

```diff
@@ -1,3 +1,9 @@
+:root {
+  --color-granite: #252629;
+  --color-nemo: #ff4a11;
+  --color-foam: #fff;
+}
```

Comment:
Consider using a custom property for the `border-radius` values (e.g., `--border-radius-small: 4px`), as this value is repeated in multiple places (e.g., figure, a, mark). 

This approach enhances maintainability and ensures consistent styling. 🎉